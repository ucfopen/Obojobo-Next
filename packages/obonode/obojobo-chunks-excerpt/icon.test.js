import React from 'react'
import renderer from 'react-test-renderer'

import Icon from './icon'

describe('Code Icon', () => {
	test('Icon', () => {
		const component = renderer.create(<Icon />)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
