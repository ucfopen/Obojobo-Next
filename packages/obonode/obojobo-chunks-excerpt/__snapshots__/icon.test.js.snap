// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Code Icon Icon 1`] = `
<svg
  viewBox="0 0 32 32"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    opacity="0.12"
  >
    <rect
      className="fill"
      height="21.58"
      rx="1.09"
      ry="1.09"
      width="22.12"
      x="4.94"
      y="5.23"
    />
  </g>
  <line
    className="stroke"
    strokeLinecap="round"
    strokeMiterlimit="10"
    x1="13.37"
    x2="18.37"
    y1="16.76"
    y2="16.76"
  />
  <line
    className="stroke"
    strokeLinecap="round"
    strokeMiterlimit="10"
    x1="8.11"
    x2="11.64"
    y1="15.76"
    y2="12.23"
  />
  <line
    className="stroke"
    strokeLinecap="round"
    strokeMiterlimit="10"
    x1="8.11"
    x2="11.64"
    y1="8.69"
    y2="12.23"
  />
</svg>
`;
